src/code/audio/audio_seqplayer.c

```diff
@@ -1,73 +1,2314 @@
+/**
+ * @file audio_seqplayer.c
+ *
+ * Manages audio sequence players, interprets and executes the Music Macro Language for Sequences
+ *
+ * Music Macro Lanuage (MML):
+ *   - A customized assembly language based on Midi
```

Comment:
```suggestion
 *   - A customized assembly language based on MIDI
```
This language gets parsed in software, does that count as "assembly" (especially since it wasnt converted to human readable text, which is what I always thought "assembly" was)? I've been calling it bytecode, but not sure what the correct terminology would be for a language that only exists at a higher level, even if its byte encoded.